clang/lib/DPCT/QueryApiMapping.cpp

```diff
@@ -0,0 +1,32 @@
+//===--------------- QueryApiMapping.cpp ----------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "QueryApiMapping.h"
+
+namespace clang {
+namespace dpct {
+
+std::unordered_map<std::string, std::shared_ptr<ApiMappingEntry>>
+    ApiMappingEntry::EntryMap;
+
+void ApiMappingEntry::registerEntry(std::shared_ptr<ApiMappingEntry> Entry) {
+  EntryMap[Entry->CudaName] = Entry;
+  EntryMap[Entry->SyclName] = Entry;
+}
+
+void ApiMappingEntry::initEntryMap() {
+#define REGISTER_ENTRY(CUDA, SYCL, DESC)                                       \

```

Comment:
updat the name of CUDA/SYCL to  SOURCE/DEST

Reply:
done